GUI/test/javafxgui/javafxcomponents/GUIBoardDisplayerTest.java

```diff
@@ -0,0 +1,14 @@
+package javafxgui.javafxcomponents;
+
+import jttt.core.boarddisplayer.BoardDisplayer;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class GUIBoardDisplayerTest {
+    @Test
+    public void displayerIsAKnownKindOfDisplayer() {
+        BoardDisplayer boardDisplayer = new GUIBoardDisplayer();
+        assertEquals(true, boardDisplayer instanceof BoardDisplayer);
```

Comment:
yep.  Nothing going on in this class yet.. it's more of a placeholder till I can move GUI Board 'display' functionality into it.